apptweak/android.py

```diff
@@ -42,7 +42,10 @@ def rankings(self, application_id, params):
 
     @classmethod
     def reviews(self, application_id, params = {}):
-        raise Exception('Not implemented for this plateform')
+        Validation().params(params)
+        required = ['country']
```

Comment:
Thanks to your PR, we noticed we had an issue on our documentation, that was since fixed: https://www.apptweak.io/documentation/android/application_reviews_all
The required param for this endpoint should be the language, not the country. :)